README.md

```diff
@@ -79,16 +79,13 @@ the working group does not automatically result in membership in these
 teams. These teams are:
 
 * Releasers team
-* LTS team
 * CITGM team
```

Comment:
I don't know if we should remove this as well or not... it seems like the maintenance of CITGM isn't just relasers anymore but not 100%